src/game/models/mod.rs

```diff
@@ -36,6 +36,16 @@ pub trait Positioned {
     }
 
     fn pos(&self) -> vector::Vector;
+
+    fn dx(&self) -> f64 {
```

Comment:
We'd only need to inherit if there are types that need to only `impl` the `Positioned` trait, but not all of the extra movement related `Kinetic` stuff like `dx`, `dy`, etc. I can't think of anything off hand.